FordLabs/peoplemover
# Add time on product cypress test

## Issue
Resolves N/A

This test would have caught a bug I manually found on the `Time on Product` page awhile back (and already fixed) where you could not open the person modal for the unassigned person.

## What was done
- [x] Write time on product cypress test

## How to test
1. This test should ensure that you can open the person modal for each person on the `Time on Product` page.
